### ISSUE TITLE
List of safe/unsafe website pages managed in bilara

### ISSUE THREAD

**Issue by ccronje** [suttacentral/bilara-data#1672]
## unsafe pages (check when safe)

- [ ] a-new-beginning_translation-[lang]-site *Delete?*
- [x] about_translation-[lang]-site
- [x] acknowledgments_translation-[lang]-site
- [ ] downloads_translation-[lang]-site *Already deleted?*
- [x] #2780
- [x] home_translation-[lang]-site
- [x] interface_translation-[lang]-site
- [ ] names_translation-[lang]-site
- [ ] numbering_translation-[lang]-site
- [x] similes_translation-[lang]-site

**Comment by noeismet** [comment]
Hello, I see that acknowledgments_translation-[lang]-site is ticked as safe here, but in the gsheet, it shows as unsafe. What should I use as a source of truth?

**Comment by ccronje** [comment]
Hi @noeismet this issue is updated first by @thesunshade. I just updated the spreadsheet but perhaps we can delete it and refer to this issue as source of truth instead.

**Comment by noeismet** [comment]
> Hi @noeismet this issue is updated first by @thesunshade. I just updated the spreadsheet but perhaps we can delete it and refer to this issue as source of truth instead.

Yes, I guess keeping only this issue should be enough and simplify it for you to maintain.

**Comment by ccronje** [comment]
Update: Spreadsheet deleted and link to it removed from original post.

**Comment by noeismet** [comment]
@ccronje , I have been in the process of translating sn-guide-sujato_translation-fr-site for a while now and suddently I wonder if I should stop? Is this one safe?

**Comment by sabbamitta** [comment]
Noé, I have recently completed the German translation of `sn-guide-sujato`, and I think it's generally safe, except for a couple of smaller things:
- One is a problem with one of the ToC items which is missing from the list at the top of the page.
- The other one is a problem with transforming Nilakkhana, but this seems to be a general problem that affects all `site` pages, at least in German, but not in English.

As far as I know, Hongda is supposed to look into these two issues.

**Comment by noeismet** [comment]
Fantastic, thank you @sabbamitta for such a prompt response, much appreciated, so I can keep going 😊

**Comment by noeismet** [comment]
Hello @ccronje, I am wondering about the status of a few site pages that are still untranslated in French but don't show in the safe/unsafe list, they are: 
- an-introduction-bodhi_translation-fr-site
- licensing_translation-fr-site
- subjects_translation-fr-site
- terminology_translation-fr-site

**Comment by sabbamitta** [comment]
Hi Noé,

I think `an-introduction-bodhi_translation-fr-site` should be safe, it's Bhikkhu Bodhi's essay, and I don't think it is subject to any changes on the side of SC. I've made the German translation a while ago.

`licensing_translation-fr-site` too has been translated into German, a long while ago, but I am not sure if it's still up-to-date.

The two index files are very sure full of mistakes and errors, given what I have seen with the other indexes that I have translated.

So probably Bhikkhu Bodhi's essay would be a good point to start at this moment.

**Comment by noeismet** [comment]
Excellent, thank you @sabbamitta 🙏

**Comment by ccronje** [comment]
@thesunshade can you please confirm the [unchecked unsafe pages](https://github.com/suttacentral/bilara-data/issues/1672#issue-1275676340) are the only ones to hold off on?

**Comment by noeismet** [comment]
Hello @ccronje, @thesunshade. There are still a bunch of pages that I could translate, but it looks like they are still currently unsafe, is that correct? 

![image](https://github.com/suttacentral/bilara-data/assets/102351746/48d8f79a-ff84-4cd0-b066-83f7e528a16c)

**Comment by thesunshade** [comment]
@ccronje and @noeismet So sorry not to respond. Somehow I had blocked notifications on this issue.

I believe that `a-new-beginning_translation-[lang]-site` is slated for deletion. Downloads appears to already be deleted.

I went ahead and checked `general-guide-sujato_translation-[lang]-site` because I believe that Bhante agreed there is nothing there to update. It's quite polished and doesn't have any timely material.

`names_translation-[lang]-site` and `similes_translation-[lang]-site` are both index pages. I believe that Bhante has no interest in developing these until a completely new index can be created. So I don't know if that means they are ready to be translated (since they won't ever be fixed) or that they shouldn't be translated (since they are rather broken).

As far as `numbering_translation-[lang]-site`, I know it needs work simply because it makes refrence to the Pootle system that is no longer used. I have to check my notes to see what is going on there.

**Comment by noeismet** [comment]
Thank you for your response @thesunshade. From what you say above, I think that only `general-guide-sujato_translation-[lang]-site` is safe for translation, should we still confirm with Bhante?

**Comment by noeismet** [comment]
Hello, my apologies for creating https://github.com/suttacentral/bilara-data/issues/2780, I hoped that I could choose one of my repo to plan the translation.

**Comment by noeismet** [comment]
> `names_translation-[lang]-site` and `similes_translation-[lang]-site` are both index pages. I believe that Bhante has no interest in developing these until a completely new index can be created. So I don't know if that means they are ready to be translated (since they won't ever be fixed) or that they shouldn't be translated (since they are rather broken).

Bhante @sujato, is it worth translating these into French at this stage, or should I wait for new ones in English to be published?
